localstack/services/apigateway/context.py

```diff
@@ -131,3 +131,7 @@ def is_websocket_request(self):
     def is_v1(self):
         """Whether this is an API Gateway v1 request"""
         return self.apigw_version == ApiGatewayVersion.V1
+
+    def cookies(self):
+        cookies = self.headers.get("cookie") or ""
```

Comment:
Another small nitpick: Can we adjust the logic to avoid returning `[""]` if the Cookie header is empty?

Reply:
sure, no problem!